core/jvm/src/main/scala/scalaz/zio/App.scala

```diff
@@ -42,6 +42,7 @@ trait App extends RTS {
   /**
    * The Scala main function, intended to be called only by the Scala runtime.
    */
+  // $COVERAGE-OFF$ Bootstrap to `Unit`
```

Comment:
Is there any other way to do this than these unsightly comments?

Reply:
The only alternative is to put these into the build definition. But then you don't really have a good way of providing reasoning. I generally prefer the comments because of that.

Reply:
I want to delete coverage entirely because I'm not using it and I don't think it's all that helpful (without mutation testing or something). Do you have any objections?

Reply:
Not really but I've used it to find parts of code that are untested. (Especially if I am looking to rely on it, see the `Duration` changes we've made)

Reply:
Sure, we do can that, in a follow-up :) Let's get this stuff in first..